runelite-client/src/main/java/net/runelite/client/plugins/jewelrycount/JewelryCountOverlay.java

```diff
@@ -114,6 +109,37 @@ public Dimension render(Graphics2D graphics)
 		return null;
 	}
 
+	private WidgetItem[] getInventoryItems()
```

Comment:
This method is duplicated in both overlays, but I wasn't sure how or where to abstract it.

Reply:
Chuck it in an Inventory class or a Utils class